src/module/system/check/check.ts

```diff
@@ -637,6 +677,9 @@ class CheckPF2e {
         // Remove the buttons if this is the discarded roll
         if (isOld) element.querySelector(".message-buttons")?.remove();
 
+        // Add mythic reroll class to dice formula element if necessary
+        if (resource === "mythic-points") element.querySelector(".dice-formula")?.classList.add("mythic");
```

Comment:
Given that you're using mythic to handle visibility, what are your thoughts on instead including hidden by default and omitting it if its mythic instead. This will show all old re-rolled formulas, but that's not really that bad, and it is more semantic.